BAZProjectC1/Movie.swift

```diff
@@ -11,3 +11,8 @@ struct Movie {
     let title: String
     let poster_path: String
```

Comment:
Nombremos las variables con lowerCammelCase 
```suggestion
    let posterPath: String
```